### PR TITLE
updating all maps for a mirror

### PR DESCRIPTION
**Pull request by Pyko1**

### CODE REVIEW

File: code/game/objects/structures/artstuff.dm [hunk 1]
```diff
@@ -66,6 +65,15 @@ GLOBAL_LIST_INIT(globalBlankCanvases, new(AMT_OF_CANVASES))
...
 
+//HEY YOU
+//ARE YOU READING THE CODE FOR CANVASES?
+//ARE YOU AWARE THEY CRASH HALF THE SERVER WHEN SOMEONE DRAWS ON THEM...
+//...AND NOBODY CAN FIGURE OUT WHY?
+//THEN GO ON BRAVE TRAVELER
+//TRY TO FIX THEM AND REMOVE THIS CODE
+/obj/item/canvas/Initialize()
+	..()
+	return INITIALIZE_HINT_QDEL //Delete on creation
```
Comment (praisenarsie) [inline code/game/objects/structures/artstuff.dm:76]: :crying_cat_face:

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -366,6 +366,15 @@
 	begin_month = JUNE
 	begin_weekday = SUNDAY
 
+/datum/holiday/moth
+	name = "Moth Week"
```
Comment (praisenarsie) [inline code/modules/holiday/holidays.dm:370]: wot in tarnation

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -1145,6 +1145,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 //////////////////
 
 /datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+//hippie edit - martial arts check because this was never implemented. sorry not sorry
+	if(attacker_style && attacker_style.help_act(user,target)) // 29/07/18, somehow this was removed but now we added it back in :)
+		return 1
+//hippie edit end
```
Comment (praisenarsie) [inline code/modules/mob/living/carbon/human/species.dm:1151]: no comment

**Comment by Carbonhell** [comment]
this pr is broken, just remake it completely, you only need to reset the tg station maps to tg version

**Pull request closed without merging**